.travis.yml

```diff
@@ -2,12 +2,16 @@ sudo: required
 language: generic
 services: docker
 env:
-- HHVM_VERSION=4.3-latest
+- HHVM_VERSION=4.8-latest
+- HHVM_VERSION=4.9-latest
+- HHVM_VERSION=4.10-latest
+- HHVM_VERSION=4.11-latest
+- HHVM_VERSION=4.12-latest
+- HHVM_VERSION=4.13-latest
```

Comment:
we don't do this because of matinenance pain - we want whatever's in composer.json + latest + nightly